### ISSUE TITLE
`web_tokenizer`: Unknown error on ",;" sequence.

### ISSUE THREAD

**Issue by geovedi** [fnl/segtok#8]
Hi,

I have this text containing ",;" sequence and `web_tokenizer` give no response or giving up/timeout. 

``` python

In [13]: text = 'https://en.wikipedia.org/wiki/Pope_Boniface_IX Pope Boniface IX (,; c. 1350 – 1 October 1404), born Piero Tomacelli, was  Pope https://en.wikipedia.org/wiki/Pope  from 2 November 1389 to his death in 1404.'
In [14]: web_tokenizer(text)
...
(no response)
```

My current workaround is by removing that sequence before doing tokenisation.

**Comment by fnl** [comment]
Hi Jim,

Thanks for the bug report! I think I see what the issue is and hope to have a fix soon.

F

**Comment by fnl** [comment]
fixed in b2adaf1dbe7d62033a776d6c8a12d73a4bb8c63a and version bumped to 1.3.1; please update your package

**Comment by geovedi** [comment]
excellent!

thanks for the update.
